src/inject.ts

```diff
@@ -5,19 +5,61 @@ const extensionID = script?.getAttribute("data-id") ?? "";
 
 const theWindow = window as any;
 
+let isJsMonoClient = false;
+
 let instance = theWindow.__STATSIG_JS_SDK__?.instance;
 if (!instance) {
   instance = theWindow.__STATSIG_SDK__?.instance;
 }
+if (!instance) {
```

Comment:
this is too nested. Please extract some helper functions and use early returns to flatten it